src/devices/bus/hp_hil/hlemouse.cpp

```diff
@@ -0,0 +1,127 @@
+// license:BSD-3-Clause
+// copyright-holders:Sven Schnelle
+#include "emu.h"
+#include "hlekbd.h"
+
+#include "machine/keyboard.ipp"
```

Comment:
A mouse shouldn't need the implementation of the `device_matrix_keyboard_interface` template...